modules/user_data/templates/06_cloudwatch_setup.sh.tpl

```diff
@@ -12,17 +12,26 @@ echo "#################################"
 echo "#    Cloudwatch Provisioning    #"
 echo "#################################"
 
-# Parse the CW Agent Config from SSM Parameter store and put it in file
-CWAGENT_CONFIG=$(aws ssm get-parameter --name "/CWAgent/Config" --query "Parameter.Value" --output text)
-echo "$CWAGENT_CONFIG" > /etc/graphdb/cloudwatch-agent-config.json
+parameters=$(aws ssm describe-parameters --query "Parameters[].Name" --output text)
 
-GRAPHDB_ADMIN_PASSWORD=$(aws --cli-connect-timeout 300 ssm get-parameter --region ${region} --name "/${name}/graphdb/admin_password" --with-decryption --query "Parameter.Value" --output text)
+# Appends configuration overrides to graphdb.properties
+if [[ $parameters == *"/CWAgent/Config"* ]]; then
+  # Parse the CW Agent Config from SSM Parameter store and put it in file
+  CWAGENT_CONFIG=$(aws ssm get-parameter --name "/CWAgent/Config" --query "Parameter.Value" --output text)
```

Comment:
shouldn't this be prefixed with the name like the other params - `/${name}/CWAgent/Config`?

Reply:
We will fix that with another pull request if you are okay with that.

Reply:
Let's do it now.